src/sql/tests/querymodel/basic

```diff
@@ -0,0 +1,145 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+build
+select;
+----
+digraph G {
+    compound = true
+    labeljust = l
+    label="select;
```

Comment:
Is this newline expected/necessary?

Reply:
It is part of the string that we get from datadriven's parser. I could/will just trim the string in the `dot` generator.